docs/step-by-step/step-00.md

```diff
@@ -0,0 +1,94 @@
+# The step-by-step Netdata tutorial
+
+Welcome to Netdata! We're glad you're interested in our health monitoring and performance troubleshooting system.
+
+Because Netdata is entirely open-source software, you can use it free of charge, whether you want to monitor one or ten
+thousand systems! All our code is hosted on [GitHub](https://github.com/netdata/netdata).
+
+This tutorial is designed to help you understand what Netdata is, what it's capable of, and how it'll help you make
+faster and more informed decisions about the health and performance of your systems and applications. If you're
+completely new to Netdata, or have never tried health monitoring/performance troubleshooting systems before, this
+tutorial is perfect for you.
+
+If you have monitoring experience, or would rather get straight into configuring Netdata to your needs, you can jump
+straight into code and configurations with our [getting started guide](../getting-started.md).
+
+> This tutorial contains instructions for Netdata installed on a Linux system. Many of the instructions will work on
+> other supported operating systems, like FreeBSD and MacOS, but we can't make any guarantees.
+
+## Before we get started
```

Comment:
It maybe could be useful somewhere in this section to let users know how and where to reach out to ask questions or raise issues if they run into them. I'm almost thinking in terms of newbie type questions people might be too afraid to raise as issues in the repo. Not really sure how we want them to engage but just thinking somewhere around here could be a good chance to communicate to them the ways they can ask questions, get help, and or otherwise get involved in the community.

Reply:
I wish we had a more user-friendly platform for people to ask questions, but GitHub is it for now. I am willing to sacrifice my own email account for people who have questions, though...